src/lib/CombatCalc.ts

```diff
@@ -216,7 +216,15 @@ export default class CombatCalc {
    */
   private wearing(item: string | string[]): boolean {
     if (Array.isArray(item)) {
-      return (item as string[]).some((i) => this.allEquippedItems.includes(i));
+      return (item as string[]).some((i) => this.allEquippedItems.find((i2) => {
+        const currentItemAndVersion = i2.split('#');
+        if (i.split('#').length === 1) {
+          // Provided item name does not have a version, simply find any version of this item
+          return i === currentItemAndVersion[0];
+        }
+        // Provided item name does have a version, find this exact match
+        return i === i2;
+      }));
```

Comment:
This doesn't account for when item isn't an array, but I think I'd rather just check versions in-line when they're necessary rather than place more logic in here.